### PR TITLE
Fix minification failures and delete unused assets

### PR DESCRIPTION
**Pull request by joelverhagen**
- Add tests to detect minification failures in the future. Basically WebGrease uses an older view the JS and CSS syntax. I had to update to the latest WebGrase to address CSS minification failures.
- Revert newer JavaScript syntax that breaks minification
- Minify some unminified files
- Delete an unused copy of jQuery UI
- Delete unused syntax highlighter CSS

Address https://github.com/NuGet/Engineering/issues/3107

### CODE REVIEW

File: tests/NuGetGallery.FunctionalTests/StaticAssets/StaticAssetsTests.cs [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+            .Select(x => new object[] { x });
+            
+
+        public HttpClient HttpClient { get; }
+
+        [Theory]
+        [Priority(2)]
+        [Category("P2Tests")]
+        [MemberData(nameof(AssetData))]
+        public async Task AllAssetsExistOnTheirOn(string assetPath)
```
Comment (loic-sharma) [inline tests/NuGetGallery.FunctionalTests/StaticAssets/StaticAssetsTests.cs:126]:
```suggestion
        public async Task AllAssetsExistOnTheirOwn(string assetPath)
```

File: tests/NuGetGallery.FunctionalTests/StaticAssets/StaticAssetsTests.cs [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+
+        private static readonly HashSet<string> BundleInputPaths = new HashSet<string>(Bundles.SelectMany(x => x.Value));
+
+        public static IEnumerable<object[]> AssetData => AssetPaths.Value.Select(x => new object[] { x });
+        public static IEnumerable<object[]> BundleOutputData => Bundles.Select(x => new object[] { x.Key });
+        public static IEnumerable<object[]> BundleInputExceptBundleOutputData => BundleInputPaths
+            .Except(Bundles.Keys.Select(GetUnMinPath))
+            .Select(x => new object[] { x });
+            
+
```
Comment (loic-sharma) [inline tests/NuGetGallery.FunctionalTests/StaticAssets/StaticAssetsTests.cs:119]: Nit: extra empty line here

File: src/NuGetGallery/Scripts/gallery/clamp.js [hunk 3]
```diff
@@ -13,7 +13,8 @@
     define([], factory);
   } else if (typeof exports === 'object') {
     // Node, CommonJS-like
-    module.exports = factory();
+    // NuGetGallery edit: commented out because we are in the browser and this line breaks WebGrease.
+    // module.exports = factory();
```
Comment (agr) [inline src/NuGetGallery/Scripts/gallery/clamp.js:17]: Remove that line completely?

Comment (joelverhagen) [reply src/NuGetGallery/Scripts/gallery/clamp.js:17]: It will be removed in minification. I'd rather notate the original since it's 3rd party.

**Pull request merged**